docs/querying/sql-functions.md

```diff
@@ -2473,28 +2575,92 @@ Shifts a timestamp forwards or backwards by a given number of time units.
 
 ## TIMESTAMP_TO_MILLIS
 
-`TIMESTAMP_TO_MILLIS(<TIMESTAMP>)`
+Returns the number of milliseconds since epoch for the given timestamp.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `TIMESTAMP_TO_MILLIS(timestamp_expr)`
+* **Function type:** Scalar, date and time
+
+<details><summary>Example</summary>
+
+The following example converts the `__time` timestamp column from the `taxi-trips` datasource into milliseconds since epoch.
+
+```sql
+SELECT 
+  "__time" AS "original_time",
+  TIMESTAMP_TO_MILLIS("__time") AS "miliseconds"
+FROM "taxi-trips"
+LIMIT 1
+```
+
+Returns the following:
+
+| `original_time` | `miliseconds` |
+| -- | -- |
+| `2013-08-01T08:14:37.000Z` | `1375344877000` |
+
+</details>
+
+
+[Learn more](sql-scalar.md#date-and-time-functions)
 
-Returns the number of milliseconds since epoch for the given timestamp.
 
 ## TIMESTAMPADD
 
-`TIMESTAMPADD(<unit>, <count>, <TIMESTAMP>)`
+Adds `count` number of a `unit` to a given timestamp.
```

Comment:
I think the original description was clearer. Maybe something like this "Adds a `count` number of  time `unit` to a given timestamp?"

Reply:
How about "Adds a `unit` of time to `timestamp` a total of `count` times."